vnext/Shared/MemoryMappedBuffer.cpp

```diff
@@ -0,0 +1,154 @@
+#include "pch.h"
+#include "MemoryMappedBuffer.h"
+
+#include <werapi.h>
+#include <windows.h>
+
+namespace {
+
+class MemoryMappedBuffer : public facebook::jsi::Buffer {
+ public:
+  MemoryMappedBuffer(const wchar_t *const filename, uint32_t offset);
```

Comment:
>const wchar_t *const [](start = 21, length = 20)

(nit) why not just const whar_t*?

Reply:
The extra const makes sure that filename cannot be modified within the constructor function. It's just a slightly stronger const guarantee and I personally prefer to use it unless there is a reason not to.

---
In reply to: [320882711](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r320882711) [](ancestors = 320882711)